ironfish/src/wallet/walletdb/multisigKeys.ts

```diff
@@ -82,7 +86,7 @@ export function isSignerMultisig(multisigKeys: MultisigKeys): multisigKeys is Mu
 export function isHardwareSignerMultisig(
   multisigKeys: MultisigKeys,
 ): multisigKeys is MultisigHardwareSigner {
-  return 'identity' in multisigKeys
+  return 'identity' in multisigKeys && !('secret' in multisigKeys)
```

Comment:
Will this come back as true for view only standard multisig?

Reply:
The standard multisig account will have the `secret` stored with its multisig keys, so this would come back as false

Reply:
This is a bug that exists previously but this should also be checking for `publicKeyPackage`.

Reply:
Why should this check for `publicKeyPackage`? Since all variations of the `MultisigKeys` type have `publicKeyPackage` it doesn't help to discriminate between them

Reply:
If you don't it's possible to think this is a MultisigHardwareSigner but it's possible that another type that has identity, does not have secret, and has a different field than public key package will pass these checks and incorrectly be labeled a hardware signer. If someone adds a type like that, they'll need to track down this code as the cause of the bug.

Reply:
Updated so that the discriminator functions each check all fields.